source/docs/06-neumes.xml

```diff
@@ -244,7 +244,7 @@
                   </figure>
                </p>
                <p>A further complication is that while the music encoding aims to narrow down and capture the meaning of the neumes in a logical and coherent system, occasionally the significance of some neumes is under debate (<abbr>e.g.</abbr>, <gi scheme="MEI">quilisma</gi>) and, despite its aim for accuracy, the encoding must remain open for future interpretations. From all these challenges has arisen the need for an early music encoding standardisation, that is, a set of rules that work for the description of any neume across all early notations regardless of the different methodologies applied to the study of individual notations and their idiosyncrasies.</p>
-               <p>Broadly speaking, Western early notations belong to two categories. On one side we have notations where two or more notes were represented by a single pen-stroke, while on the other side there are notations where the notes are graphically separated by means of discrete dots or short pen-strokes. These distinctions have been described even within single notational styles as interrupted neumes `&lt;ineume&gt;` or uninterrupted `&lt;uneume&gt;` (Morent &amp; Schräder 2008; Morent 2011; MEI Guidelines v3, ch. 6: Neume Notation), and now as gapped or not gapped (Behrendt, Bain, &amp; Helsen 2017; MEI Guidelines v4, ch. 6: Neume Notation). To date, the MEI Neumes Module has been tested mainly on stroke notations (St. Gall, Old Hispanic, etc.), but also on Aquitanian point-notation.</p>
+               <p>Broadly speaking, Western early notations belong to two main categories. On one side we have notations where two or more notes were represented by a single pen-stroke, while on the other side there are notations where the notes are graphically separated by means of discrete dots or short pen-strokes. These distinctions have been described even within single notational styles as gapped or not gapped (Behrendt, Bain, &amp; Helsen 2017). To date, the MEI Neumes Module has been tested mainly on square notations and stroke notations (St. Gall, Old Hispanic, etc.), but also on Aquitanian point-notation.</p>
```

Comment:
```suggestion
               <p>Broadly speaking, Western neume notations belong to two main categories. On one side we have notations where two or more notes were represented by a single pen-stroke, while on the other side there are notations where the notes are graphically separated by means of discrete dots or short pen-strokes. These distinctions have been described even within single notational styles as gapped or not gapped (Behrendt, Bain, &amp; Helsen 2017). To date, the MEI Neumes Module has been tested mainly on square notations and stroke notations (St. Gall, Old Hispanic, etc.), but also on Aquitanian point-notation.</p>
```

Reply:
But what about the concept of gapped or not gapped?

Reply:
Perhaps a sentence pointing out how to record the difference in MEI? Is there a `@gap` attribute? Which of the notations mentioned at the end are "gapped"? Which are "not gapped"?

I think this paragraph is lacking the "so what" part... It's OK that some notations are defined as gapped or ungapped, but what's the specific impact on MEI so that this is worth mentioning to the readers?

Reply:
I would not remove it. 

There is some valuable information about the two different main categories of early notations, and that the current encoding approach (as described in the given literature incl. the guidelines) is mostly tested with stroke and square notations, right?

Examples for encodings of different types are given in the following section 6.4.

But I agree with @ahankinson that maybe one or two examples at the end of this paragraph (or references to existing examples) would help to illustrate the concept of gapped/not gapped for the uninformed reader.

Reply:
Yes, I totally see your point that after the recent changes the content of the paragraph appears quite "gapped". 

> Are all of the notations mentioned gapped, or ungapped? Or does the list represent a set of neume notations that represent both gapped and ungapped styles?

As far as I understand the concept, there is not a single neume notational style that is one OR the other, but they all include gapped or ungapped neumes to different extents (e.g., in St. Gall notation Clivis, Torculus and Porrectus would be examples of ungapped neumes, climacus and scandicus for gapped ones).

> save for one caption this is the only mention of 'gapped' or 'ungapped' in the whole chapter

> [musicEnfanthen:] maybe one or two examples at the end of this paragraph (or references to existing examples) would help to illustrate the concept of gapped/not gapped for the uninformed reader.

I previously missed that there is indeed a prominent example right at the beginning of the chapter in section [6.1.1 Basic four elements](https://music-encoding.org/guidelines/dev/content/neumes.html#basicfourelements) that illustrates the concept, but here called connected vs. non-connected (Fig. 58 & 59).

So while I agree that there are some inconsistencies, I wonder if fixing that wouldn't require a broader review that goes beyond the scope of this PR (and then possibly beyond version 5)?

Reply:
I propose to move this discussion to a new issue for MEI 5.x and get this PR done

Reply:
please see #1251